php/plan.sh

```diff
@@ -38,11 +39,7 @@ pkg_include_dirs=(include)
 pkg_interpreters=(bin/php)
 
 do_build() {
-  # The configuration scripts unset LD_RUN_PATH when testing linking for configured options,
-  # so the resulting 'conftest' binaries cannot run due to being unable to find libstdc++.so
-  # This allows those binaries to execute while limiting the scope of LD_LIBRARY_PATH
-  # to the execution of `./configure`.
-  LD_LIBRARY_PATH="$(pkg_path_for gcc)/lib" ./configure --prefix="${pkg_prefix}" \
+  ./configure --prefix="${pkg_prefix}" \
```

Comment:
this will break libzip support!
```suggestion
  LD_LIBRARY_PATH="$(pkg_path_for gcc)/lib" ./configure --prefix="${pkg_prefix}" \
```

Reply:
I wrote the comment, and it's wrong 😞.  The underlying issue was that `core/gcc-libs` wasn't in the `pkg_deps` therefore the `conftest` binaries were unable to execute,  not that the path was being manipulated.

The above LD_LIBRARY_PATH modifies only the _runtime_ for `./configure` and has no effect on commands executed later in the build, or in the runtime for the php package.  Since this change adds `core/gcc-libs` to the `pkg_deps`, `./configure` is able complete successfully without setting LD_LIBRARY_PATH,  so LD_LIBRARY_PATH and the comment can safely be removed. 

I've put together a small test case to validate.  Note:  I'm not a php dev, so suggestions on correctness are welcome 😄 

Given the following php:
```test.php
<?
  $za = new ZipArchive;
  $za->open('test.zip',ZipArchive::CREATE|ZipArchive::OVERWRITE);
  $za->addFile("php/plan.sh");
  $za->close();
?>
```

When executing it under core/php/7.2.8/20181008145855 (current stable):
```shell
[31][default:/src:0]# hab pkg exec core/php/7.2.8/20181008145855 php test.php

Fatal error: Uncaught Error: Class 'ZipArchive' not found in /src/test.php:2
Stack trace:
#0 {main}
  thrown in /src/test.php on line 2
```

When executed it on a build with these changes:
```shell
[32][default:/src:0]# hab pkg exec smacfarlane/php php test.php
[33][default:/src:0]# unzip -l test.zip
Archive:  test.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
     2027  2018-10-30 14:45   php/plan.sh
---------                     -------
     2027                     1 file
```

Reply:
🎉 👍 💯 🥇

Reply:
@smacfarlane How about adding these steps to a test we can include? That would prevent any zip regression in the future.

Reply:
@predominant Done!

@universam1 Does the above address your concerns?

Reply:
Cool, I’ll try!

Reply:
@universam1  I'm going to merge this,  please open an issue if the resulting packages do not contain libzip support.